_includes/js-config.html

```diff
@@ -0,0 +1,23 @@
+  <script>
```

Comment:
Is `picc` defined somewhere else already? I'd expect something like this just to be safe:

```js
picc = picc || {};
```